compiler/rustc_middle/src/ty/generic_args.rs

```diff
@@ -250,17 +250,17 @@ impl<'tcx> GenericArg<'tcx> {
     }
 
     #[inline]
-    pub fn as_type(self) -> Option<Ty<'tcx>> {
+    pub fn as_region(self) -> Option<ty::Region<'tcx>> {
```

Comment:
why did you flip these lol

Reply:
region, type, const is the correct order :<

Reply:
definitely do not agree. ty region const is the order in folders.